.github/workflows/nightly-test-integrations.yml

```diff
@@ -309,13 +309,107 @@ jobs:
         run: datadog-ci junit upload --service "$GITHUB_REPOSITORY" $TEST_RESULTS_DIR/results.xml
 
 
+  peering_commontopo-integration-test:
+    runs-on: ${{ fromJSON(needs.setup.outputs.compute-xl) }}
```

Comment:
This 1.16.x test runs on large runner [here](https://github.com/hashicorp/consul/blob/0ce3ec94a2650a39601ef63c755a74c2412d7326/.github/workflows/nightly-test-integrations-1.16.x.yml#L317), perhaps this should be changed to large as well?